### PR TITLE
rust-mode 0.3.0

### PR DESCRIPTION
**Pull request by ilovezfs**

### CODE REVIEW

File: Formula/rust-mode.rb [hunk 1]
```diff
@@ -3,6 +3,8 @@
 class RustMode < EmacsFormula
   desc "Emacs major mode for editing Rust code"
   homepage "https://github.com/rust-lang/rust-mode"
+  url "https://github.com/rust-lang/rust-mode/archive/0.3.0.tar.gz"
+  sha256 "6ec0c86eb9a19b850d6e4a42db56f80ff9586cc2013d8b2f9a652fc5f0e7be45"
   head "https://github.com/rust-lang/rust-mode.git"
 
   depends_on :emacs => "23.1"
```
Comment (rwhogg) [inline Formula/rust-mode.rb]: I think we should increase the dependency to Emacs 24. Support for Emacs 23 was removed [here](https://github.com/rust-lang/rust-mode/pull/142).

Comment (rwhogg) [inline Formula/rust-mode.rb]: This should also fix the error on Travis (which apparently installs Emacs 23.3.1 by default):

You must install the `ert` dependency; see README.md

Comment (ilovezfs) [inline Formula/rust-mode.rb]: Is that `homebrew/emacs/ert-runner`?

Comment (rwhogg) [inline Formula/rust-mode.rb]: No, I don't believe it is. `ert-runner`'s description says it's a tool for projects tested using ERT, not ERT itself.

**Comment by rwhogg** [comment]
Thanks @ilovezfs :smile_cat:

**Pull request closed without merging**